File: Aditya_c/calculator.c
#include <stdio.h>
int main()
{
    int option;
    float sum, difference, product, ratio;
    char l[2][10] = {"first", "second"};
    float a[2];
    for (int i = 0; i < 2; i++)
    {
        printf("Enter %s number - ", l[i]);
        scanf("%f", &a[i]);
    }
    printf("Enter your option- Add(1) / Substract(2) / multiply(3) / divide(4) - ");
    scanf("%d",&option);
    switch (option)
    {
    case 1:
    {
        sum = a[0]+a[1] ;
        printf("Sum = %.2f\n",sum);
        break;
    }
    case 2:
        difference = a[0]-a[1];
        printf("Difference = %.2f\n",difference);
        break;
    case 3:
        product = a[0]*a[1];
        printf("Product = %.2f\n",product);
        break;
    case 4:
        ratio = a[0]/a[1];
        printf("Ratio = %.2f\n",ratio);
        break;
    default:
        printf("Enter a valid number\n");
        break;
    }
    return 0;
}

File: python_/projects/calculator.py
def takinginputs():
    num1=float(input("Enter the first number:"))
    num2=float(input("Enter the second number:"))
    return num1,num2
def options():
    print("--------Simple calculator:--------")
    print("Enter options:")
    print("1.Addition")
    print("2.Substraction")
    print("3.Multiplication")
    print("4.Division")
    option=int(input("Enter your option for which you want to perform your task:"))
    return option
def calculation(option,num1,num2):
    if option == 0 or option > 4:
        print("Enter a valid number!")
    else:
        if option==1:
            sum = num1+num2
            print("Sum=",sum)
        elif option==2:
            difference= num1-num2  
            print("Difference=",difference)
        elif option==3:
            product=num1*num2   
            print("Product=",product)
        else:
            ratio=num1/num2
            print("Division=",ratio)
option=options()
num1, num2=takinginputs()
calculation(option,num1,num2)


